VideoLocker/res/values/strings.xml

```diff
@@ -183,7 +183,7 @@
     <!-- Label for the menu popup for viewing the course on the web -->
     <string name="course_view_on_web">View on Web</string>
     <!-- Label for the menu popup for changing mode to Videos Only/Full Course -->
-    <string name="course_change_mode">Mode picker</string>
+    <string name="course_change_mode">Course mode picker</string>
```

Comment:
@BenjiLee After consulting with @cptvitamin we arrived at "Course viewing options".
(I got a good reminder that we shouldn't include the object name (e.g. menu, button) in the screen reader text because if that is appended, the screen reader user will hear "course viewing options menu menu".)